### DISCUSSION TITLE
Automazione con sonda temperatura

### DISCUSSION THREAD

**Discussion by earlwasp** [indomus/forum#2486] [HUB: Home Assistant]
Salve a tutti, 
ho un solare termico con delle tapparelle che dovrebbero chiudersi al raggiungimento di una certa temperatura dell'acqua, e riaprirsi quando cala di un tot di gradi (es.: temp arriva a 60° chiudi, se scende sotto i 55° riapri).
Ho integrato gli shelly in HA, vedo la temperatura e riesco ad inviare i comandi alle tapparelle per apri/chiudi, ma sono bloccato sulle automazioni. 
Qualcuno mi può dare qualche dritta ? 

Grazie

**Comment by 50494554524F** [comment]
devi usare questi trigger... above e below

https://www.home-assistant.io/docs/automation/trigger/#numeric-state-trigger
